*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:36505 (IPv4)
Extension host agent listening on 36505

[08:38:26] 




[08:38:26] Extension host agent started.
[08:38:26] [127.0.0.1][1a1b2aa6][ManagementConnection] New connection established.
[08:38:26] [127.0.0.1][b002565d][ExtensionHostConnection] New connection established.
[08:38:26] Deleted uninstalled extension from disk ms-python.vscode-pylance /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2024.3.2
[08:38:26] [127.0.0.1][b002565d][ExtensionHostConnection] <1069> Launched Extension Host Process.
[08:38:33] Getting Manifest... ms-python.debugpy
[08:38:33] Getting Manifest... ms-python.python
[08:38:33] Installing extension: ms-python.debugpy
[08:38:33] Installing extension: ms-python.python
[08:38:34] Getting Manifest... ms-python.python
[08:38:35] Waiting for already requested installing extension ms-python.python ms-python.debugpy
[08:38:43] Extension signature is verified: ms-python.debugpy
[08:38:44] Extension signature is verified: ms-python.python
[08:38:47] Extracted extension to file:///home/pi/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-armhf: ms-python.debugpy
[08:38:47] Renamed to /home/pi/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-armhf
[08:38:47] Extracting extension completed. ms-python.debugpy
[08:38:47] Marked extension as uninstalled ms-python.debugpy-2024.2.0-linux-armhf
[08:38:53] Extracted extension to file:///home/pi/.vscode-server/extensions/ms-python.python-2024.4.1: ms-python.python
[08:38:56] Renamed to /home/pi/.vscode-server/extensions/ms-python.python-2024.4.1
[08:38:56] Extracting extension completed. ms-python.python
[08:38:56] Extension installed successfully: ms-python.python
[08:38:56] Finished waiting for already requested installing extension ms-python.python ms-python.debugpy
[08:38:56] Extension installed successfully: ms-python.debugpy
[08:38:56] Marked extension as uninstalled ms-python.python-2024.2.1
[08:39:52] [127.0.0.1][1a1b2aa6][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[08:39:52] [127.0.0.1][b002565d][ExtensionHostConnection] <1069> Extension Host Process exited with code: 0, signal: null.
Cancelling previous shutdown timeout
[08:39:52] Cancelling previous shutdown timeout
Last EH closed, waiting before shutting down
[08:39:52] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[08:41:34] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[08:41:34] Cancelling previous shutdown timeout
[08:41:34] [127.0.0.1][75b00e44][ManagementConnection] New connection established.
[08:41:34] [127.0.0.1][2a7c458d][ExtensionHostConnection] New connection established.
[08:41:34] [127.0.0.1][2a7c458d][ExtensionHostConnection] <1291> Launched Extension Host Process.
rejected promise not handled within 1 second: CodeExpectedError: Could not find pty on pty host
stack trace: CodeExpectedError: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:43:32578)
    at D.updateTitle (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:43:27710)
    at T.<computed> (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:43:22925)
    at Object.call (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:34:5061)
    at u.s (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:32:5165)
    at u.q (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:32:4667)
    at l.value (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:32:4065)
    at m.y (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at m.fire (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process._ (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:25:10025)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
[08:41:37] Error [CodeExpectedError]: Could not find pty on pty host
    at D.U (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:43:32578)
    at D.updateTitle (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:43:27710)
    at T.<computed> (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:43:22925)
    at Object.call (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:34:5061)
    at u.s (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:32:5165)
    at u.q (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:32:4667)
    at l.value (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:32:4065)
    at m.y (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:27:1906)
    at m.fire (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:27:2123)
    at process._ (/home/pi/.vscode-server/bin/863d2581ecda6849923a2118d93a088b0745d9d6/out/vs/platform/terminal/node/ptyHostMain.js:25:10025)
    at process.emit (node:events:514:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
New EH opened, aborting shutdown
[08:46:34] New EH opened, aborting shutdown
